.github/actions/waffles/requirements.txt

```diff
@@ -1,4 +1,5 @@
 docopt==0.6.2
 Flask==2.0.3
 markupsafe==2.0.1
-git+https://github.com/cds-snc/notifier-utils.git@48.3.0#egg=notifications-utils
+# git+https://github.com/cds-snc/notifier-utils.git@48.3.0#egg=notifications-utils
+git+https://github.com/cds-snc/notifier-utils.git@python-3.10#egg=notifications-utils
```

Comment:
I guess we will need a follow up PR to point this to a tagged version of utils? Also - this won't work if the `python-3.10` branch is auto deleted when this PR is merged. So maybe we should temporarily turn the "auto delete branches" feature off?

Reply:
good point. I'll do it all in this PR I think, ie change waffles to point to 50.0.0 and then tag quickly after the merge. Should work!